cypress/e2e/about.cy.ts

```diff
@@ -0,0 +1,77 @@
+describe('About page', () => {
+    context('Desktop resolution', () => {
+        beforeEach(() => {
+            cy.viewport(1920, 1080)
+            cy.visit('/about')
+        })
+
+        it('shows the desktop nav', () => {
+            cy.get('[data-testid="desktop-nav"]').should('exist')
+        });
+
+        it('does not show the hamburger component', () => {
+            cy.get('[data-testid="hamburger-component"]').should('not.exist')
+        });
+
+        it('has a heading', () => {
+            cy.get('[data-testid="about-heading"]').contains('About Find a Doc, Japan')
+        });
+
+        it('has a subheading', () => {
+            cy.get('[data-testid="about-subheading"]').contains('Connecting people in Japan to the healthcare services they need, in the languages they need.')
+        });
```

Comment:
This is from when I was writing tests a few months ago. @ermish said we should avoid hard coding text into the tests. I do not know a better way. Can we import the JSON we are using and use template literals. This way any small change that could happen to that text file is automatically updated and checked in both places?